tests/test_http_request.py

```diff
@@ -198,10 +197,9 @@ def test_empty_formdata(self):
         r1 = self.request_class("http://www.example.com", formdata={})
         self.assertEqual(r1.body, b'')
 
-    @unittest.skipUnless(six.PY2, "TODO")
     def test_default_encoding(self):
         # using default encoding (utf-8)
-        data = {'one': 'two', 'price': '\xc2\xa3 100'}
+        data = {b'one': b'two', b'price': b'\xc2\xa3 100'}
```

Comment:
@eliasdorneles , done in https://github.com/redapple/scrapy/commit/f5a1e2905eb5bd4df4cea13a6ca21e8d422b9772